backstop/tests/menu.html

```diff
@@ -29,162 +40,1915 @@
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m6.00263 4.80681v6.39946a.27938.27938 0 0 0 .44292.24059l3.45479-3.17567a.34022.34022 0 0 0 .00046-.4807l-.00046-.00044-3.45479-3.22382a.269.269 0 0 0 -.44292.24058z"
+            />
+          </svg>
+        </li>
+        <li
+          class="iui-menu-item iui-large"
+          tabindex="0"
+        >
+          <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+            />
+          </svg>
+          <span class="iui-label">
+            <div class="iui-menu-label">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et
+              dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+              ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+              fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+              mollit anim id est laborum.</div>
+            <div class="iui-menu-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+              ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+              voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+              proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+          </span>
+          <!-- Use caret-right-small.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+          >
+            <path
+              d="m6.00263 4.80681v6.39946a.27938.27938 0 0 0 .44292.24059l3.45479-3.17567a.34022.34022 0 0 0 .00046-.4807l-.00046-.00044-3.45479-3.22382a.269.269 0 0 0 -.44292.24058z"
+            />
+          </svg>
+        </li>
+        <li
+          class="iui-menu-divider"
+          aria-hidden="true"
+        ></li>
+        <li
+          class="iui-menu-item"
+          tabindex="0"
+        >
+          <span class="iui-label">
+            <div class="iui-menu-label">My projects</div>
+          </span>
+        </li>
+      </ul>
+
+    </section>
+
+    <hr />
+
+    <h2>In context</h2>
+    <h3>Header</h3>
```

Comment:
Does not align.
![image](https://user-images.githubusercontent.com/36186912/124443013-fa9e4f80-dd85-11eb-869b-7abc037d3b93.png)

Reply:
I can't figure out why the test images have the menu's not aligned properly with the header buttons.  They look fine in the demo before:
![Screen Shot 2021-07-07 at 4 37 26 PM](https://user-images.githubusercontent.com/849817/124825467-ab9d1980-df41-11eb-8a77-7f1fc4be1685.png)

They menus are placed using `margin-left` inline and then some random value to make it look like they align.

Reply:
For me it is not only in tests but also when I open HTML

Reply:
Fixed menu placement, for some reason margin should change based on screen size.